packages/Uploader/src/components/File/File.js

```diff
@@ -47,14 +52,8 @@ function File({ error, fileKey, name, progress, size, status }) {
           <Popover isDark isEager>
             <Popover.Tip />
             <Popover.Trigger>
-              <Button.Icon
-                kind={Button.Icon.types.kind.MINOR}
-                onClick={() => {
-                  restartFileUpload(fileKey);
-                }}
-                size={Button.Icon.types.size.SMALL}
-              >
-                <RetryIcon />
+              <Button.Icon kind={Button.Icon.types.kind.MINOR} onClick={() => {}}>
+                <InfoCircle />
```

Comment:
Maybe this should be the caution icon and be red like the error message?  The black is quite eye catching.

Reply:
![Screen Shot 2020-12-15 at 4 20 25 PM](https://user-images.githubusercontent.com/23224777/102288509-7ec7dd00-3ef1-11eb-81db-6c5398578169.png)

Reply:
It should at least be `tokens.textColor.icon`, as that black is quite stark.

Reply:
actually if you cancel the file why would you want to retry? now second thoughts we shouldn't even have an icon, and the infoIcon is actually correct isn't it?

cc @sunminsarahlee

Reply:
> actually if you cancel the file why would you want to retry?

yeah that's true

Reply:
![Screen Shot 2020-12-16 at 11 36 29 AM](https://user-images.githubusercontent.com/19418590/102397637-fbf75e80-3f92-11eb-9cd4-23e95f9c234b.png)

like this

Reply:
I am okay with removing `i` icon for cancelling action.